README.md

```diff
@@ -1,6 +1,6 @@
 # RunCat_for_windows
 
-A cute running cat animation on your windows taskbar.
+### A cute running cat animation on your windows taskbar.
```

Comment:
If your aim is to emphasize the passages, use `*passage*` instead of `###`.
This applies to the other lines using `###`.